scripts/decredStakeVoting.md

```diff
@@ -3,45 +3,44 @@
+In rare cases, it is possible for votes to be missed even though they were cast. This can happen when miners are processing blocks faster than the votes can propagate through the network. This leads to the mining of blocks with less than 5 votes. If a block is mined with less than the required 5 votes, the remaining votes destined for that block are considered ‘missed’ and the funds used to purchase them are unlocked.
 
-A ticket may also cast votes concerning other agenda items put up for a community vote. One example of an item that was approved in this manner was a consensus rule upgrade improving the algorithm which sets ticket price.
+After a stake voter's ticket has been drawn and their votes are successfully cast, the Decred used for the ticket is unlocked and a reward for voting, called a ticket reward, is deposited into the ticket owners account. This ticket reward amounts to 6% of the total Decred block reward. After a waiting period of 20 hours, these funds become available for spending.
 
-In rare cases, it is possible for votes to be missed even though they were cast. This can happen when miners are processing blocks faster than the votes can propagate through the network. This leads to the mining of blocks with less than 5 votes. If a block is mined with less than the required 5 votes, the remaining votes destined for that block are considered ‘missed’ and the funds used to purchase them are unlocked.
+This equation can be used to calculate the annual percentage rate of returns gained through ticket rewards:
 
-After a stake voter's ticket has been drawn and their votes are successfully cast, the Decred spent on the ticket is unlocked and a reward for voting, called a ticket reward, is deposited into the ticket owners account. This ticket reward amounts to 6% of the total Decred block reward. After a waiting period of 20 hours, these funds become available for spending.
+`(ticket reward / ticket price) / ticket lifespan [in years] = Annual Percentage Rate`
 
-Stake voting requires no action by the stakeholder other than purchasing tickets. By default, tickets are configured to `abstain` from voting on current agenda items, however, `yes` or `no` votes can be configured easily through the Decrediton staking wallet.
+Assuming, for example, a ticket reward of 1.3 Decred, a ticket price of 88 Decred, with the average ticket lifespan of 30 days (or 0.082 years), the annual percentage rate of 12 tickets purchased in a year works out to 18%.
 
-Decrediton uses cryptographic ticket "receipts" to verify and allow stake voters to interface with Politeia; Decred's chain-anchored proposal system. Through Politeia, stake voters can participate in Decred self-governance by proposing issues for the community to vote on and voting on those proposed by others.
+Best of all, that 18% comes risk free; staked coins never leave your wallet and are always unlocked after a ticket expires, votes, or misses.
```

Comment:
Current calculations show about 15% APR after other variables are included, such as falling reward cost over time, so probably best to stick with that.
May not be technically accurate to talk about leaving the wallet since they are locked and dont physically exist.